engine/classes/ElggPlugin.php

```diff
@@ -851,9 +851,6 @@ public function start($flags) {
 			throw new Exception($this->getError());
 		}
 
-		// Preload static config file
-		$this->getStaticConfig('');
-
```

Comment:
@hypeJunction you put this here in https://github.com/Elgg/Elgg/commit/c289b291bbc45f0f18178c823e1b7ff870298def#diff-f4771ef801aa02abca099036f4493824R854, any particular reason?

Reply:
I think it was related to integraiton test bootstrap, but since tests are green, I don't think there is any strict order.